include/nuanceur/builder/Operations.h

```diff
@@ -133,6 +137,9 @@ namespace Nuanceur
 
 	CUintRvalue Load(const CArrayUintValue& buffer, const CIntValue& index);
 	void Store(const CArrayUintValue& buffer, const CIntValue& index, const CUintValue&);
+	void Store(const CArrayUshortValue& buffer, const CIntValue& index, const CUshortValue&);
+	void Store(const CArrayUcharValue& buffer, const CIntValue& index, const CUshortValue&);
```

Comment:
Is there still a use to this overload? I guess it works but only the lower 8-bit of a value would be stored inside the array? It seems to me it could be misleading and not very useful.

Reply:
![image](https://user-images.githubusercontent.com/5013823/113445329-974efb00-93ed-11eb-9a75-28fae5198867.png)
this was a direct 16bit write using that store (as i believe array is only used for traversing, atleast with spirv/nvidia), having said that, i do recall this causing an issue, so ill remove it now, since its not used anyway.

Reply:
Interesting, I wonder how that works on the hardware level. Especially the unaligned write... I always thought unaligned writes was something that was restricted/special because it has to be implemented using two partial writes. 

On a GPU where everything is running in parallel, I wonder how they can guarantee things to be coherent. But then maybe that's why it was causing issues.

Anyways, yeah, better leave it out for now.